adback-solution-to-adblock/includes/class-ad-back-activator.php

```diff
@@ -37,6 +37,21 @@ public static function activate()
 
         $charset_collate = $wpdb->get_charset_collate();
 
+        $fields = [
+            'email' => get_bloginfo('admin_email'),
+            'website' => get_site_url(),
+        ];
+        $response = Ad_Back_Generic::postContents('https://adback.co/tokenoauth/register/en', $fields);
```

Comment:
I think the treat of the request (and this response) must be in Ad_Back_Generic.